tests/dummy/app/styles/app.scss

```diff
@@ -1,4 +1,3 @@
-@import "wheely";
```

Comment:
This line was deleted because i've deleted node_modules/frontend-basis from dependencies